app/load_data/connect_db.py

```diff
@@ -0,0 +1,22 @@
+import os
+import requests
+import sys
+from sqlalchemy import create_engine, Column, Integer, String, ForeignKey
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker, relationship
+from load_data.models import Base
+
+
+def main():
+    # Connect to DB
+    sys.stderr.write("Connecting to the DB")
+    engine = create_engine(os.environ.get('DB_URI', '../../sqlite:///drugs.db'))
```

Comment:
See comment in readme, we should explore pathlib for generating this uri and likely a config file